src/main/java/it/gov/pagopa/common/web/exception/ServiceException.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.common.web.exception;
+
+
+import lombok.Getter;
+
+@Getter
+public class ServiceException extends RuntimeException {
+    private final String code;
+    private final boolean printStackTrace;
+    private final transient ServiceExceptionResponse response;
```

Comment:
Let's remove the transient keyword
Let's make the ServiceExceptionResponse interface to extend Serializable
```suggestion
    private final ServiceExceptionResponse response;
```